README.md

```diff
@@ -1242,6 +1243,94 @@ Here is a complete example for copying bundles from `prod` to `integration`.
    The `--map-version` option adds a specific duration to the version of each
    copied file and bundle. Run `python scripts/copy_bundles --help` for details.
 
+## 7.4 Debugging running lambdas via Pycharm
```

Comment:
Move the meat of this to the docstring of RemoteDebugSession and just leave pointer here.

Reply:
I did this and converted the documentation to RST since that's the convention used by Sphinx.